Makefile

```diff
@@ -156,6 +195,48 @@ define proto_compile
 
 endef
 
+.PHONY: setup-scripts
+setup-scripts: 
+	rm -rf /tmp/jaeger	&&	rm -rf ./scripts
+	git clone --depth 1 https://github.com/jaegertracing/jaeger.git /tmp/jaeger
```

Comment:
how many scripts do we need? We should only download what's needed. And it's faster to get those scripts via curl from github